notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb

```diff
@@ -0,0 +1,597 @@
+    "    config[\"metric\"] = metric_config\n",
+    "    config[\"algorithms\"] = algorithms\n",
+    "    \n",
+    "    return config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "608279bb",
+   "metadata": {
+    "id": "FAGmlKQ83ecE"
+   },
+   "source": [
+    "### Run Quantization Pipeline and Accuracy Verification\n",
+    "\n",
+    "The following 9 steps show how to quantize the model using the POT API. The optimized model and collected min-max values will be saved. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "738cb5e7",
+   "metadata": {
+    "id": "QTOoQnSetzQM",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\" Download dataset and set config\n",
+    "\"\"\"\n",
+    "config = get_config()  \n",
+    "init_logger(level='INFO')\n",
+    "logger = get_logger(__name__)\n",
+    "save_dir = increment_path(Path(\"./yolov5/yolov5m/yolov5m_openvino_model/\"), exist_ok=True)  # increment run\n",
+    "save_dir.mkdir(parents=True, exist_ok=True)  # make dir\n",
+    "\n",
+    "# Step 1: Load the model.\n",
+    "model = load_model(config[\"model\"])\n",
+    "\n",
+    "# Step 2: Initialize the data loader.\n",
+    "data_loader = YOLOv5DataLoader(config[\"dataset\"])\n",
+    "\n",
+    "# Step 3 (Optional. Required for AccuracyAwareQuantization): Initialize the metric.\n",
+    "metric = COCOMetric(config[\"metric\"])\n",
+    "\n",
+    "# Step 4: Initialize the engine for metric calculation and statistics collection.\n",
+    "engine = IEEngine(config=config[\"engine\"], data_loader=data_loader, metric=metric)\n",
+    "\n",
+    "# Step 5: Create a pipeline of compression algorithms.\n",
+    "pipeline = create_pipeline(config[\"algorithms\"], engine)\n",
+    "\n",
+    "metric_results = None\n",
+    "\n",
+    "# Check the FP32 model accuracy.\n",
+    "metric_results_fp32 = pipeline.evaluate(model)\n",
+    "\n",
+    "logger.info(\"FP32 model metric_results: {}\".format(metric_results_fp32))\n",
+    "\n",
+    "# Step 6: Execute the pipeline to calculate Min-Max value\n",
+    "compressed_model = pipeline.run(model)\n",
+    "\n",
+    "# Step 7 (Optional):  Compress model weights to quantized precision\n",
+    "#                     in order to reduce the size of final .bin file.\n",
+    "compress_model_weights(compressed_model)\n",
+    "\n",
+    "# Step 8: Save the compressed model to the desired path.\n",
+    "optimized_save_dir = Path(save_dir).joinpath(\"optimized\")\n",
+    "save_model(compressed_model, Path(Path.cwd()).joinpath(optimized_save_dir), config[\"model\"][\"model_name\"])\n",
+    "\n",
+    "# Step 9 (Optional): Evaluate the compressed model. Print the results.\n",
+    "metric_results_i8 = pipeline.evaluate(compressed_model)\n",
+    "\n",
+    "logger.info(\"Save quantized model in {}\".format(optimized_save_dir))\n",
+    "logger.info(\"Quantized INT8 model metric_results: {}\".format(metric_results_i8))"
+   ]
```

Comment:
![image](https://user-images.githubusercontent.com/10940214/179021317-ae421db6-4ce3-4468-a678-fcbdc4e16298.png)
I got this error after running this cell.

Reply:
Your error shows cannot find model xml file. My demo require to download Yolov5 from other repo at 1st cell, then convert pytorch model to FP32 IR. If you cannot find model xml, please check have you run all cell from beginning, and if you download and convert model correctly. BTW, you need to update your python env by updated requirements.txt.

Reply:
This is running well now. Thanks, @FionaZZ92. We should include in the markdown cell that this process should take a few minutes. In that case, they would be aware of the processing time. Is it possible to include which step is creating which output? It could be great to have that in the output of this specific cell.

![image](https://user-images.githubusercontent.com/10940214/179510127-43d97d4a-cf7e-4178-a2db-2f1f177fe66b.png)

Reply:
fixed. Please have a check, thx.